juicy-html.html

```diff
@@ -118,7 +161,16 @@
         JuicyHTMLPrototype.isAttached = false;
         JuicyHTMLPrototype.attachedCallback = function() {
             this.isAttached = true;
-            this.loadTemplate_();
+            // autostamp
+            // translate legacy `content` attribute
+            if(this.hasAttribute('content')){
+                warnAboutDeprecatedContentOn(this);
+                forwardContentValue(this, this.getAttribute('content'));
+            } else {
+                // TODO(tomalec): read pre-upgrade properties in cheap manner
+                this.hasAttribute('href') && this.attributeChangedCallback('href', null, this.getAttribute('href'));
+                this.hasAttribute('html') && this.attributeChangedCallback('html', null, this.getAttribute('html'));
+            }
```

Comment:
Replace the above if/else with just:

```js
this.hasAttribute('href') && this.attributeChangedCallback('href', null, this.getAttribute('href'));
this.hasAttribute('html') && this.attributeChangedCallback('html', null, this.getAttribute('html'));
this.hasAttribute('content') && this.attributeChangedCallback('content', null, this.getAttribute('content'));
```

Reply:
For the `<template is="juicy-html" href="smth.html" content="foo">` It will set and remove attributes like crazy, triggering a lot of confusing events. But I agree it makes slightly DRYier code. The question is what is important. Performance and less hard to debug behavior for really ugly usage (border case, but still achievable case) or DRY maintenance of 2 lines until we drop/remove `content` support.

Reply:
On the other hand, attributes `href` and `html` unset each other to null, so having both `href` and `html` also unsets attributes as crazy. Maybe we should not do unsetting in the attributes?

Reply:
> Maybe we should not do unsetting in the attributes?

Then the end result of such unordered attribute setting would lead to unpredictable behavior.

I'd rather split it into two separate custom elements, but I'd say it's a step beyond the scope of this PR. Also, I'd prefer to do it in CE v1 only.